docs/getting-started.md

```diff
@@ -79,6 +79,7 @@ If you want to add yourself to this list, let us know!
 
 ## Deploying Thanos
 
+* [WIP] Detailed, free, in-browser interactive tutorial [as Katacoda Thanos Course](https://killercoda.com/thanos/)
```

Comment:
So Killercode or Katacoda? :smile: I think this needs to be:
```suggestion
* [WIP] Detailed, free, in-browser interactive tutorial [as Killercoda Thanos Course](https://killercoda.com/thanos/)
```

Reply:
I had to leave at least one old reference to make sure reviewers are paying attention :detective:, thanks!